docs/usage/tables.md

```diff
@@ -1,44 +1,72 @@
 Tables
 ======
 
-A Sleeper instance contains one or more tables. Each table has four important
-properties: a name, a schema for storing data for that table, a state store for 
-storing metadata about the table, and a flag to denote whether the table is 
-online or not (see [here](../design.md#Tables) for more information about 
-online tables). All resources for the instance, such as the S3 bucket used for 
-storing data in a table, ECS clusters and lambda functions are shared across 
-all the tables.
+A Sleeper instance contains one or more tables. Each table has four important properties: a name, a schema for storing
+data for that table, a state store for storing metadata about the table, and a flag to denote whether the table is
+online or not. See the [design documentation](../design.md#Tables) for more information about online tables.
+
+All resources for the instance, such as the S3 bucket used for storing data in a table, ECS clusters and lambda
+functions are shared across all the tables.
 
 ## The metadata store
 
-Each table has metadata associated to it. This metadata is stored in a
-state store and consists of information about files that are in the 
-system, and the partitions. See the [original design document](design/transaction-log-state-store.md)
-for the current state store backed by a transaction log.
+Each table has metadata associated to it. This metadata is stored in a state store and consists of information about
+files that are in the system, and the partitions. See the [design documentation](../design.md#State_store) for more
+information.
 
 ## Add/edit a table
 
-Scripts can be used to add, rename and delete tables in a Sleeper instance.
+Scripts can be used to add, rename and delete tables in a Sleeper instance. If using the scripts, creating a new table
+will consist of the following steps:
+
+1. Use the `estimateSplitPoints.sh` script to estimate split points from your data.
+2. Use the `addTable.sh` script to create the table.
+3. Use the `reinitialiseTable.sh` script with the split points from the first step.
+4. Use the `sendToIngestBatcher.sh` script to send your data to the ingest batcher to be added to the table. See
+   the [ingest documentation](ingest.md#ingest-batcher) for how to use this.
+
+All of these scripts will rely on a schema for your table, which should be created first.
+See [creating a schema](schema.md) for how to set up a schema for your table.
+
```

Comment:
Can we also add a complete example that can be copied and pasted? e.g.

    cat schema.json
      {
    "rowKeyFields": [ 
      {
        "name": "key",
        "type": "StringType"
      }
    ],
    "valueFields": [
      {
        "name": "value",
        "type": "StringType"
      }
    ]
    }
    ID=mysleeperid
    ./scripts/utility/estimateSplitPoints.sh schema.json 128 32768 splits.file mybucket/file.parquet
    ./scripts/utility/addTable.sh $ID table1
    ./scripts/utility/reinitialiseTable.sh $ID table1 true splits.file
    ./scripts/utility/sendToIngestBatcher.sh $ID table1 mybucket/file.parquet

Reply:
Done